### PR TITLE
Candidate flow

### PR DESCRIPTION
**Pull request by MichaelCherry**
Creation of skeleton for Candidate Flow through eMib Sample test as illustrated by:

image

Example of flow
image

image

image

image

image

image

### CODE REVIEW

**Comment by MichaelCherry** [comment]
image

image


image

File: frontend/src/App.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import "./App.css";
 import Status from "./Status";
 import Home from "./Home";
 import Experiment from "./Experiment";
+import E_MIB from "./components/eMIB/E_MIB";
```
Comment (caleybrock) [inline frontend/src/App.js]: Component names in JS are `TitleCase` with no underscores. 
I'd call it `Emib`? Just to not confuse it with a constant variable name.

File: frontend/src/App.js [hunk 2]
```diff
@@ -19,6 +20,7 @@ const App = () => {
           <Route exact path="/" component={Home} />
           <Route path="/experiment" component={Experiment} />
           <Route path="/status" component={Status} />
+          <Route path="/eMIB-Sample" component={E_MIB} />
```
Comment (caleybrock) [inline frontend/src/App.js]: URLs should be all lowercase, like `emib-sample`.

File: frontend/src/components/eMIB/Background.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from "react";
+
+class Background extends Component {
+  render() {
+    return (
+      <div>
+        <h2>Background Page</h2>
+        <h3>Org Chart</h3>
```
Comment (caleybrock) [inline frontend/src/components/eMIB/Background.jsx]: Nice use of proper `h` tag structure!

File: frontend/src/test/Background.test.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { shallow } from "enzyme";
+import Background from "../components/eMIB/Background";
+
+it("renders without crashing", () => {
```
Comment (caleybrock) [inline frontend/src/test/Background.test.js]: No need to include the "renders without crashing" tests anymore - they'll just take up time and will be checked anyways now that we have other tests.

File: frontend/src/components/eMIB/E_MIB.jsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+import Confirmation from "./Confirmation";
+import HowTo from "./HowTo";
+import Background from "./Background";
+import Inbox from "./Inbox";
+
+const PAGES = {
```
Comment (caleybrock) [inline frontend/src/components/eMIB/E_MIB.jsx]: Love this use of enums!!

File: frontend/src/test/E_MIB.test.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  const div = document.createElement("div");
+  ReactDOM.render(<E_MIB />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+it("renders initial message", () => {
+  const wrapper = shallow(<E_MIB />);
+  const initialMessage = <p>Welcome to the eMIB Sample Test</p>;
+  expect(wrapper.contains(initialMessage)).toEqual(true);
+});
```
Comment (caleybrock) [inline frontend/src/test/E_MIB.test.js]: It's be great to have tests when you force a state change in your test, and then check that the proper sub-component displays. For example, if state is `howTo`, does the `HowTo` component display?

Here's an example: https://stackoverflow.com/questions/45598423/react-enzyme-how-to-test-for-subcomponent

And here's an enzyme cheatsheet that I find useful: https://devhints.io/enzyme

**Comment by fnormand01** [comment]
It could be useful to put the following diagram in the description:

image

Also explain in one line what we are trying to do here. What do you think?

**Comment by fnormand01** [comment]
Other than your new tests you'll create (suggested by @caleybrock), everything else is good with me, since we built that together. I tested it on my side and everything works well.

**Pull request merged**